api/types/plugin.go

```diff
@@ -391,6 +405,14 @@ func (s *PluginJamfSettings) CheckAndSetDefaults() error {
 	return nil
 }
 
+func (s *PluginJiraSettings) CheckAndSetDefaults() error {
+	if s.ServerUrl == "" {
```

Comment:
Is this the only configuration parameter that's required? How about project ID and others?